food-truck-api/src/main/java/food/truck/api/truck/Truck.java

```diff
@@ -0,0 +1,41 @@
+package food.truck.api.truck;
+
+import lombok.Data;
+
+import javax.persistence.*;
+
+@Data
+@Entity
+@Table(name = "truck")
+public class Truck {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "truck_id", nullable = false)
+    Long id;
```

Comment:
Curious about why this is a `Long` not a `long`, since Long (object form) is more expensive than a long (primitive). Does Hibernate require it or something like that?

Reply:
I'm not too sure whether Hibernate requires it or not. I had been using the `User` class as a reference and it had it implemented that way. @KevinDeMars was there a specific reason?

Reply:
That's left over from the template project. I don't see a reason to have it as a wrapper object, but I'm not very familiar with Spring's ORM.